mlflow/utils/model_utils.py

```diff
@@ -166,16 +166,20 @@ def _validate_path_exists(path, name):
         )
 
 
-def _validate_and_copy_model_code_and_config_paths(code_path, config_path, path):
+def _validate_and_copy_model_code_and_config_paths(
+    code_path: str, config_path: Optional[str], path: str
+):
     """Copies the model code from code_path to a directory.
 
     Args:
         code_path: A file containing model code that should be logged as an artifact.
         config_path: A file containing model config code that should be logged as an artifact.
         path: The local model path.
     """
-    _validate_path_exists(code_path, "code")
-    _validate_path_exists(config_path, "config")
+    if code_path:
+        _validate_path_exists(code_path, "code")
```

Comment:
Do we need this `if` check if its not option in the `_validate_and_copy_model_code_and_config_paths` signature?